libr/lang/p/cpipe.c

```diff
@@ -38,13 +37,17 @@ static int lang_cpipe_file(RLang *lang, const char *file) {
 		cc = strdup ("gcc");
 	}
 	char *buf = r_str_newf ("%s %s -o %s/bin%s"
```

Comment:
can you use the shell escape function to avoid code injection issues? r2 can be installed on paths with spaces which may make this line fail